dashboard/app/models/callout.rb

```diff
@@ -41,7 +41,7 @@ def self.find_or_create_all_from_tsv!(filename)
   def self.first_or_create_from_tsv_row!(row_data)
     id_or_name = row_data[CSV_HEADERS[:script_id]]
 
-    unless id_or_name.to_i != 0
+    unless id_or_name.to_i.to_s == id_or_name.to_s
```

Comment:
i wish i had thought of this like 10 minutes ago to suggest it, but we probably shouldn't allow IDs to be used in callouts.tsv at all. we should be able to rename the `script_id` column to `script_name` and always look up by name. 

your change here is safer, but i think we'll need to disallow IDs completely in seeding logic

Reply:
> i think we'll need to disallow IDs completely in seeding logic

Completely agree! I didn't do that because we have [a fixture](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/test/fixtures/callouts.tsv) that still uses ids and I'm not sure the best way to fix it...

Reply:
Actually, I think it's just used in [callout_test.rb](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/test/models/callout_test.rb). I could try to update that test in this PR as well but it might be better to wait and do that as a follow-up (I'm happy to own). Thoughts?

Reply:
let's do it in a follow-up just to keep this fix targeted